File: modul3/Python/PRAK304-2310817210023-RifkyPutraMahardika.py
nilai = int(input("Input : "))
if nilai == 0:
    print("Output : Nol")
elif nilai > 0 and nilai < 10:
    print("Output : Satuan")
elif nilai > 9 and nilai < 20:
    print("Output : Belasan")
elif nilai > 19 and nilai < 100:
    print("Output : Puluhan")
else:
    print("Output : Anda Menginput Melebihi Limit Bilangan")

File: modul3/C/PRAK304-2310817210023-RifkyPutraMahardika.c
#include<stdio.h>
int main(){
    int nilai;
    printf("Input : ");
    scanf("%d", &nilai);

    if(nilai == 0){
        printf("Output : Nol");}
    else if(nilai > 0 && nilai < 10){
        printf("Output : Satuan");}
    else if(nilai > 9 && nilai < 20){
        printf("Output : Belasan");}
    else if(nilai > 19 && nilai < 100){
        printf("Output : Puluhan");}
    else{
        printf("Output : Anda Menginput Melebihi Limit Bilangan");}
return 0;
}